test-image/Dockerfile

```
@@ -1,6 +1,8 @@
 FROM node:10-slim
-ENV NODE_PORT 3000
-COPY package* ./
-RUN npm install
-COPY . .
-CMD ["npm", "start"]
+COPY package.json .
```

you should be able to do multiple on one line
```suggestion
COPY package.json yarn.lock .
```

---

sure